### PR TITLE
Remove moment

### PR DESCRIPTION
**Pull request by anehx**
Closes #1783

### CODE REVIEW

File: packages/distribution/addon/utils/inquiry-deadline.js [hunk 1]
```diff
@@ -21,13 +21,12 @@ function decorator(
       const value = inquiry.document?.deadline.edges[0]?.node.value;
       const isAnswered = inquiry.status === "COMPLETED";
 
-      const deadline = moment.utc(value);
-      const now = moment.utc();
+      const deadline = DateTime.fromISO(value).startOf("day");
+      const now = DateTime.now().startOf("day");
 
-      const isOverdue = !isAnswered && now.isAfter(deadline, "day");
+      const isOverdue = !isAnswered && now > deadline;
```
Comment (czosel) [inline packages/distribution/addon/utils/inquiry-deadline.js:27]: luxon seems to have a nice API :+1:

File: packages/form/addon/instance-initializers/setup-pikaday-i18n.js [hunk 2]
```diff
@@ -13,9 +12,17 @@ class Translations extends EmberObject {
...
+    return [];
+  }
+
+  get weekdays() {
+    return [];
+  }
+
+  get weekdaysShort() {
+    return [];
+  }
```
Comment (czosel) [inline packages/form/addon/instance-initializers/setup-pikaday-i18n.js:25]: Why do we need to keep those?

**Review by czosel — APPROVED** [review]
Just a minor question, awesome work! :100:

File: packages/form/addon/instance-initializers/setup-pikaday-i18n.js (hunk 2 above)
Comment (anehx) [reply packages/form/addon/instance-initializers/setup-pikaday-i18n.js]: This whole thing is obsolete, removed.

**Pull request merged**